appveyor.yml

```diff
@@ -27,43 +43,81 @@ install:
   - 7z x mariadb-%MARIADB_VERSION%.tar > nul
   - cd mariadb-%MARIADB_VERSION%
   - rmdir /S /Q storage\mroonga\
+  - ps: Get-Content ..\mroonga\version | ForEach-Object { $env:MROONGA_VERSION = $_ }
   - move ..\mroonga storage\mroonga
   - git clone --quiet --depth 1 --recursive https://github.com/groonga/groonga.git ..\groonga
+  - ps: Get-Content ..\groonga\base_version | ForEach-Object { $env:GROONGA_VERSION = $_ }
+  - ps: Get-Content ..\groonga\include\groonga\version.h.in | ForEach-Object { $_ -replace "@GRN_VERSION@","$env:GROONGA_VERSION" } | Set-Content ..\groonga\include\groonga\version.h
```

Comment:
This should be generated by CMake.
https://github.com/groonga/groonga/blob/master/CMakeLists.txt#L73-L75

Reply:
Fixed